tests/ert/ui_tests/cli/analysis/test_design_matrix.py

```diff
@@ -0,0 +1,60 @@
+from textwrap import dedent
+
+import numpy as np
+import pandas as pd
+import pytest
+
+from ert.config import ErtConfig
+from ert.mode_definitions import ENSEMBLE_EXPERIMENT_MODE
+from ert.storage import open_storage
+from tests.ert.ui_tests.cli.run_cli import run_cli
+
+
+@pytest.mark.usefixtures("copy_poly_case")
```

Comment:
When I try running this test locally it fails with `Worksheet named 'DefaultSheet' not found`.
Should this use fixture "copy_poly_design_case" or something instead?

Reply:
I have removed `test-data/poly_design` as we can generate this case on the fly and then it we need the matrix for experiments it can be copied from the temp dir.

Reply:
> When I try running this test locally it fails with `Worksheet named 'DefaultSheet' not found`.

This should be fixed now. Got a commit lost when rebasing but now it should be back.